sstables/sstables.cc

```diff
@@ -2659,14 +2659,14 @@ future<> sstable::close_files() {
     });
 }
 
-static inline sstring dirname(const sstring& fname) {
+static inline sstring parent_path(const sstring& fname) {
```

Comment:
no objection since `parent_path` is a better name for this function, just for the protocol, `dirname` follows the posix convention of dirname/basename.  See `man 3 dirname`

Reply:
Right. I think dirname was good too. I think the confusing name wasn't of dirname() but but of fsync_directory() which sounded like it was acting on a directory path - not its parent. But you fix that in the next patch.